extensions/adobe/experience/offer-management/tag.schema.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Tags (across XDM) are being addressed by a completely separate PR and Issue.  This can go away.

Reply:
We'll be happy to get rid of that as soon as there's a replacement we can refer to. Feel free to point out to the existing schema.

Reply:
Unfortunately, that work is not complete and will not complete for 0.9.  We will be sure to make it a priority for 1.0

Reply:
Just steal the `keywords` from `asset`.

Reply:
@trieloff If it was just that easy ...
Assets is using XMP Keywords. I hope the XMP Keywords adhere to the semantics and intentions that Tags do. Tags are mainly used to form dynamic collections of our XDM Instances (objects like an `Offer` that are tagged with `Thankgiving`). Tags also have an identity besides their human readable value. This is required to perform precise queries. 
The human readable value in a tag can change over time (typo was corrected to `Thanksgiving`). Tags will need be able to be grouped together. Another use case is to help analyze the and report the usage of instances (dimensions in a report). So high level, they are indeed like XMP keywords but the mechanics of their usage is maybe different. References to objects with a human readable representation vs a simple string with a locale.

The property xmp:keywords defines an array of structured objects but does not factor out a definition (type) for them array items. This could of course be fixed.

For now we should wait until the PR that @lrosenthol mentions comes through and then we should see what the best choice is.

Reply:
After looking into XMP Keywords a bit, I think we cannot simply push in the semantic and use cases that come with XMP; and Keywords specifically. Our object model will use the Offer Management Tags until there is something simpler but yet suitable for our use cases.